Probesys/bileto
# imp: Allow to give colors to labels

## Related issue(s)



https://github.com/Probesys/bileto/issues/283

## How to test manually



1. Create a label and give it a color different than grey
2. Associate the label to a ticket
3. Verify in the admin, in the tickets' list (including filters) and show/new/edit pages that the label is correctly colorized

## Checklist
